virttest/libvirt_cgroup.py

```diff
@@ -420,8 +420,8 @@ def get_standardized_virsh_info(self, virsh_cmd=None, virsh_dict=None):
                     # no need to check scheduler type, it's fixed for qemu
                     continue
                 if "quota" in schedinfo_item:
-                    if schedinfo_value in ["-1", "18446744073709551",
-                                           "17592186044415"]:
+                    if (schedinfo_value in ["18446744073709551", "17592186044415"]
```

Comment:
the change looks good to me, a simple explanation comment before it , is encouraged

Reply:
comment added, thx